modules/javafx.fxml/src/main/java/com/sun/javafx/fxml/ModuleHelper.java

```diff
@@ -31,16 +31,17 @@
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 
-@SuppressWarnings("removal")
 public class ModuleHelper {
     private static final Method getModuleMethod;
     private static final Method getResourceAsStreamMethod;
 
     private static final boolean verbose;
 
     static {
-        verbose = AccessController.doPrivileged((PrivilegedAction<Boolean>) () ->
```

Comment:
You can merge this assignment with the declaration on line 38. Or you can keep this so the check of verbose is in the same block with its assignment.

Reply:
I think I'll keep this one as is to minimize changes.